gen3/bin/kube-setup-fence.sh

```diff
@@ -7,6 +7,7 @@
 
 source "${GEN3_HOME}/gen3/lib/utils.sh"
 gen3_load "gen3/lib/kube-setup-init"
+namespace=$(g3kubectl config view | grep namespace: | cut -d':' -f2 | cut -d' ' -f2)
```

Comment:
remove this line

Reply:
Shouldn't we have something in `kube-setup-fence` that creates and annotates a role and links it to the `useryaml-job` service account?  Or maybe we should retire the `useryaml-job` service account and create a new `fence-serviceaccount` that links to a fence-role that also has dbgap-bucket access?  If you don't want to do that now, then create a jira to do that in the future ...